docs/cloud/overview.mdx

```diff
@@ -406,28 +406,27 @@ fluvio topic delete string-quotes
 
 ## Next Steps
 
-Now you're familiar with using InfinyOn Cloud with the `fluvio` CLI, Check out our [Guides] and [Tutorials].
+Now you're familiar with using InfinyOn Cloud with the [Cloud CLI], Check out our [Tutorials].
 
-[`fluvio profile`]: /docs/fluvio/cli/fluvio/profile
-[`Fluvio CLI`]: /docs/fluvio/cli/introduction
-[Fluvio Cluster]: /docs/fluvio/concepts/architecture/overview
+[`fluvio profile`]: fluvio/cli/fluvio/profile.mdx
```

Comment:
I don't think `profile` exists. We should port it over though. It's an important feature.

Reply:
fixed